docs/howtos/adding-a-new-component.md

```diff
@@ -13,11 +13,14 @@ advice on designing and structuring the actual Rust code, and follow the
 introduces any new dependencies.
 
 Use [UniFFI](https://mozilla.github.io/uniffi-rs/) to define how your crate's
-API will get exposed to foreign-language bindings. Place the following in your `Cargo.toml`:
+API will get exposed to foreign-language bindings. Lookup the installed uniffi
+version on other packages (eg `grep uniffi
+components/init_rust_components/Cargo.toml`) and use the same version. Place
+the following in your `Cargo.toml`:
 
 ```
 [dependencies]
-uniffi = { workspace = true }
+uniffi = { version = "0.28.2" }
```

Comment:
consider somehow noting that the version isn't stable or static - ie, I believe this will be wrong in just a matter of weeks.

Reply:
yeah, did mention this in text, but replaced the version string with a placeholder now to make it extra clear and copy/paste safe